manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-    "spark_version": "2.3.2",
+    "spark_version": "2.4.0",
     "default_spark_dist": {
         "hadoop_version": "2.7",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.3.2-bin-2.7.7.tgz"
+        "uri": "https://s3-us-west-2.amazonaws.com/infinity-artifacts/spark/spark-2.4.0-bin-2.7.7.tgz"
```

Comment:
Hm, why not `downloads.mesosphere`?

Reply:
If you still want to keep infinity then please edit link to: 
`https://infinity-artifacts.s3.amazonaws.com/spark/spark-2.4.0-bin-2.7.7.tgz`